---
title: communities
intro: Applications build on top of MOLGENIS platform
layout: black
---

Many groups and consortia have adopted MOLGENIS and configured it to their research needs. Below communities that have created MOLGENIS configurations that are highly reusable.

# Catalogues: make data and sample findable


Links:
* [BBMRI-ERC directory of biobanks](http://directory.bbmri-eric.nl) (Contact: Petr Holub, BBMRI-ERIC, Graz)
* [BBMRI-NL catalogue of biobank collections](http://catalogue.bbmri.nl) (Contact: David van Enckevort, BBMRI-NL, Groningen)
* [RD-Connect samples catalogue](http://samples.rd-connect.nl) (Contact: Mary Wong, Telethon, Milano)
* [PALGA pathology samples catalogue](http://www.palgaopenbaredatabank.nl/) (Contact: aanvraag@palga.nl)
* [LifeLines catalogue of data items](http://catalogue.lifelines.nl) (Contact: Aafje Dotinga, LifeLines)

Publications:
* Holub et al (2016) BBMRI-ERIC Directory: 515 Biobanks with Over 60 Million Biological Samples. [Biopreservation and Biobanking](https://www.ncbi.nlm.nih.gov/pubmed/27936342)
* Roxana Merino-Martinez et al (2016) Toward Global Biobank Integration by Implementation of the Minimum Information About BIobank Data Sharing (MIABIS 2.0 Core). [Biopreservation and Biobanking](https://www.ncbi.nlm.nih.gov/pubmed/26977825)

# FAIR and interoperability platform

Links:
* [SORTA recoder](http://molgenis.org/sorta)
* TODO: we have demo servers but not a very good representation. Some videos maybe?

Publications:
* Pang et al (2017) BiobankUniverse: automatic matchmaking between datasets for biobank data discovery and integration. [Bioinformatics](https://www.ncbi.nlm.nih.gov/pubmed/29036577)
* Wilkinson et al (2016) The FAIR Guiding Principles for scientific data management and stewardship. [Scientific Data](https://www.ncbi.nlm.nih.gov/pubmed/26978244)
* Pang et al (2016) MOLGENIS/connect: a system for semi-automatic integration of heterogeneous phenotype data with applications in biobanks. [Bioinformatics](https://www.ncbi.nlm.nih.gov/pubmed/27153686)
* Pang et al (2015) BiobankConnect: software to rapidly connect data elements for pooled analysis across biobanks using ontological and lexical indexing. [Journal of the Medical Informatics Association](https://www.ncbi.nlm.nih.gov/pubmed/25361575)
* Pang et al (2015,)SORTA: a system for ontology-based re-coding and technical annotation of biomedical phenotype data. [Database(Oxford)](https://www.ncbi.nlm.nih.gov/pubmed/26385205)
* Lancaster et al (2015) Cafe Variome: general-purpose software for making genotype-phenotype data discoverable in restricted or open access contexts. [Human Mutation](https://www.ncbi.nlm.nih.gov/pubmed/26224250)

# Patient / mutation registries

TODO describe rare disease patient mutation registry here.

Links:
* [Deb-central: patient registry for Epidermolosis Bullosa](http://www.deb-central.org) (Contact: Peter van den Akker, UMCG)
* [International registry of Microvillus Inclusion Disease (MVID) patients and associated MYO5B, STX3 and STXBP2 mutations](http://www.mvid-central.org)(Contact: Sven van IJzendoorn, UMCG)
* [Arrhythmogenic Right Ventricular Dysplasia/Cardiomyopathy (ARVD/C) Genetic Variants Database](http://www.arvcdatabase.info)(Contact: Paul van der Zwaag, UMCG)
* [Open-access database on CHD7 mutations](http://www.CHD7.org)(Contact: Nicole Corsten-Janssen, UMC Groningen)

Publications:
* Van den Akker et al (2011) The international dystrophic epidermolysis bullosa patient registry: an online database of dystrophic epidermolysis bullosa patients and their COL7A1 mutations. [Human Mutation](https://www.ncbi.nlm.nih.gov/pubmed/21681854)
* Lazarinne et al (2015) The ARVD/C genetic variants database: 2014 update. [Human Mutation](https://www.ncbi.nlm.nih.gov/pubmed/25676813)
* Van der Velde et al (2013) An overview and online registry of microvillus inclusion disease patients and their MYO5B mutations. [Human Mutation](https://www.ncbi.nlm.nih.gov/pubmed/24014347)
* Janssen et al (2012) Mutation update on the CHD7 gene involved in CHARGE syndrome. [Human Mutation](https://www.ncbi.nlm.nih.gov/pubmed/22461308)

# multi-omics study warehouse

HFGP, IBD, ...

Links:
* [Prospectively study of 1000 IBD patients from the Northern provinces of the Netherlands](http://1000ibd.org) (Contact: Rinse Weersma, UMC Groningen)
* [Human Functional Genomics Project](http://hfgp.bbmri.nl) (Contact: Mihai Netea, RadboudUMC Nijmegen)
* [WormQTL - Public archive and analysis web portal for natural variation data in Caenorhabditis spp.](http://www.wormqtl.org) (Contact: Jan Kammenga, Wageningen University)

Publications:
* Van der Velde et al (2018) MOLGENIS Research: Advanced bioinformatics data software for non-bioinformaticians. [Bioinformatics](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty742/5085379)
* Li et al (2016) A Functional Genomics Approach to Understand Variation in Cytokine Production in Humans. [Cell](https://www.ncbi.nlm.nih.gov/pubmed/27814507)
* Netea et al (2016) Understanding human immune function using the resources from the Human Functional Genomics Project. [Nature Medicine](https://www.ncbi.nlm.nih.gov/pubmed/27490433)
* Snoek et al (2013) WormQTL--public archive and analysis web portal for natural variation data in Caenorhabditis spp. [Nucleic Acids Research](https://www.ncbi.nlm.nih.gov/pubmed/23180786)


# DNA variant pathogenicity classification

Describe here about VKGL.

Links:
* [GAVIN variant interpretation tool](http://molgenis.org/gavin) (Contact: Joeri van der Velde)
* [INSAID consortium: Classification Database](https://molgenis77.gcc.rug.nl/) (Contact: Marielle van Gijn, UMCU)
* [Chromosome 6 Project](https://www.chromosome6.org/) (Contact: Conny van Raavenswaaij, UMCG)
* [NIPTRIC: clinical interpretation of non-invasive prenatal testing (NIPT)](http://www.niptric.eu/)
] (Contact: Lennart Johansson, UMCG)

Publications:
* Van der Velde (2017) GAVIN: Gene-Aware Variant INterpretation for medical sequencing. [Genome Biology](https://www.ncbi.nlm.nih.gov/pubmed/28093075)
* Van Gijn et al (2018) New workflow for classification of genetic variants' pathogenicity applied to hereditary recurrent fevers by the International Study Group for Systemic Autoinflammatory Diseases (INSAID).[Journal of Medical Genetics](https://www.ncbi.nlm.nih.gov/pubmed/29599418)
* Engwerda et al (2018) The phenotypic spectrum of proximal 6q deletions based on a large cohort derived from social media and literature reports. [European Journal of Human Genetics](https://www.ncbi.nlm.nih.gov/pubmed/29904178)
* Sikkema-Raddatz et al (2016) NIPTRIC: an online tool for clinical interpretation of non-invasive prenatal testing (NIPT) results. [Science reports](https://www.ncbi.nlm.nih.gov/pubmed/27917919)

# NGS data processing pipelines

DNA, RNA, microbiome....

Links:
* [NGS DNA pipeline](https://github.com/molgenis/NGS_DNA)
* [NGS RNA pipeline](https://github.com/molgenis/NGS_RNA)
* [NGS scRNA pipelin](https://github.com/molgenis/NGS_ScRNA)
* [NGS Microbiome](https://github.com/molgenis/NGS_Microbiome)
* [CoNVaDING](https://github.com/molgenis/CoNVaDING)
* [Genotype Array Pipeline](https://github.com/molgenis/GAP)

Publications:
* Kanterakis et al (2015) Molgenis-impute: imputation pipeline in a box. [BMC Research Notes](https://www.ncbi.nlm.nih.gov/pubmed/26286716)
* Deelen et al (2015) Calling genotypes from public RNA-sequencing data enables identification of genetic variants that affect gene-expression levels. [Genome Medicine](https://www.ncbi.nlm.nih.gov/pubmed/25954321)
* Johanson et al (2016) CoNVaDING: Single Exon Variation Detection in Targeted NGS Data. [Human Mutation](https://www.ncbi.nlm.nih.gov/pubmed/26864275)
* Deelen et al (2014) Genotype harmonizer: automatic strand alignment and format conversion for genotype data integration. [BMC Research Notes](https://www.ncbi.nlm.nih.gov/pubmed/25495213)
